_data/learning.yml

```diff
@@ -0,0 +1,93 @@
+# Categories group mini-books on the 'Learn' landing page
+#
+#  'fa-icon' refers to the font-awesome icon to display
+#          next to the category on the landing page
+#
+categories:
+  - name: Getting started
+    fa-icon: fas fa-cubes
+  # - name: Advanced programming methods
+  #   fa-icon: fas fa-laptop-code
+
+
+
+# Mini-books are single-page or multi-page tutorials
+#
+# 'link' refers to the 'permalink' field of the corresponding page
+#
+books:
+  - title: Quickstart Fortran Tutorial
+    description: An introduction to the Fortran syntax and its capabilities
+    category: Getting started
+    link: /learn/quickstart
+    pages:
+      - link: /learn/quickstart/hello_world
+      - link: /learn/quickstart/variables
+      - link: /learn/quickstart/arrays_strings
+      - link: /learn/quickstart/operators_control_flow
+      - link: /learn/quickstart/organising_code
+
+
+
+# Web links listed at the bottom of the 'Learn' landing page
+#
+reference-links:
+  - name: Fortran wiki
+    url: http://fortranwiki.org/
+    description: A rich collection of Fortran articles and resources in an editable wiki format
+    
+  - name: "Fortran 2018 Standard Interpretation Document"
+    url: https://j3-fortran.org/doc/year/18/18-007r1.pdf
+    description: "J3/18-007r1 F2018, specification of the base Fortran 2018 language"
+
+  - name: "J3 Fortran Proposals"
+    url: https://github.com/j3-fortran/fortran_proposals
+    description: a repository for community collaboration on proposals for the Fortran Standards Committee
+
+  - name: "J3: US Fortran Standards Committee"
+    url: https://j3-fortran.org/
+    description: J3 is the US National Body for the international Fortran standards committee
+  
+  - name: "WG5: International Fortran Standards Committe"
+    url: https://wg5-fortran.org/
+    description: 
+
+  - name: "Scivision Fortran 2018 Examples"
+    url: https://github.com/scivision/fortran2018-examples
+    description: A github repository containing code samples for various Fortran 2018 features
+
+
+
+
+# Print books listed at the bottom of the 'Learn' landing page
+#
+reference-books:
+  - author: Michael Metcalf, John Reid, and Malcolm Cohen
```

Comment:
```suggestion
  - author: Metcalf, M., Reid, J. and Cohen M.
```

Reply:
I made this suggestion so the names are formatted the same way as other entries. But you could also list full names for all books, which is actually my preference.